packages/engine/src/xr/systems/XRSystem.ts

```diff
@@ -30,7 +31,15 @@ const startXRSession = async () => {
     dispatchLocal(EngineActions.xrSession() as any)
 
     // Current WebXRManager.getCamera() typedef is incorrect
-    ;(Engine.xrManager as any).getCamera().layers.enableAll()
+    // @ts-ignore
+    const cameras = Engine.xrManager.getCamera() as ArrayCamera
+    cameras.layers.enableAll()
+    cameras.cameras.forEach((camera) => {
```

Comment:
Looks like these layers are overridden after they are set, I did put the layer setup in a `setTimeout` and the avatar appeared again.